k8s/templates/job-template.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "studio.fullname" . }}-django-config
+  labels:
+    app: {{ template "studio.fullname" . }}
+data:
+  DJANGO_SETTINGS_MODULE: {{ . Values.settings }}
+  DJANGO_LOG_FILE: /var/log/django.log
+  MPLBACKEND: PS
+  STUDIO_BETA_MODE: "yes"
+  RUN_MODE: k8s
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "studio.fullname" . }}-db-creds
+  labels:
+    app: {{ template "studio.fullname" . }}
+type: Opaque
+data:
+  DATA_DB_USER: {{ .Values.postgresql.postgresUser | default "" | b64enc }}
```

Comment:
I'm not sure why we need an additional secret here since the database information has been saved as secret in `studio-secrets.yaml`. Could you please elaborate on this? Thank you!

Reply:
studio-secrets.yaml's data entries weren't formatted for easy input into `envFrom`, so I elected to create another secret. However if you think that's a code smell, I can just use the old secret for now.

Reply:
Is it possible to use `envFrom` for the configMap and use `env` for pulling the database secrets from the secret created in `studio-secrets.yaml`? just feel like two secrets with duplicated content may not be necessary, but if using both `env` and `envFrom` is not a good way, then please feel free to keep this secret object 👍

Reply:
Sounds good on using secrets for the DB. I'll make the change.